nss-day-cohort-47/tabloidfullstack-bingo
# Hp delete posts

# Description
Wrote functionality to delete a post and any row in Comments and PostTags which is reliant on the deleted post.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Login to the app, view the home page and my posts and select delete on a post. You will be prompted to confirm your delete, if you choose to confirm the post will be deleted, if you deny the confirmation the post should not be deleted.
# Checklist:
